src/rating/rating.spec.ts

```diff
@@ -582,6 +595,42 @@ describe('ngb-rating', () => {
       expect(element.nativeElement).toHaveCssClass('ng-untouched');
     });
 
+    it('should not update template driven form by clicking disabled control', () => {
```

Comment:
Please wrap the test in `async(...)` otherwise error reporting is broken:

```sh
Chrome 80.0.3987 (Mac OS X 10.14.6) ERROR
  An error was thrown in afterAll
  Error: Expected undefined to equal 3.
```

When it should be:

```sh
Chrome 80.0.3987 (Mac OS X 10.14.6) ngb-rating forms should not update template driven form by clicking disabled control FAILED
	Error: Expected $[0] = false to equal true.
```